cmd/plan_disable.go

```diff
@@ -53,7 +53,18 @@ var PlandDisabledCmd = &cobra.Command{
 			os.Exit(-1)
 		}
 
-		util.PrintResult(p, []string{"ID", "Name", "Enabled", "Branch", "RefreshInterval"}, json)
+		util.PrintResult(
+			p,
+			[]string{"ID", "Name", "Enabled", "Branch", "RefreshInterval"},
+			map[string]string {
+				"ID": "Plan ID",
+				"Name": "Plan Name",
+				"Enabled" : "Active",
+				"Branch" : "Git Branch",
+				"RefreshInterval": "Interval",
```

Comment:
This map is for headers for rendering plans under a composite. I'm currently, keeping things consistent with how we show plans on our web app, based on data available via API.